desc/compute/_omnigenity.py

```diff
@@ -284,18 +355,69 @@ def _sqrtg_B(params, transforms, profiles, data, **kwargs):
     transforms={"B": [[0, 0, 0]]},
     profiles=[],
     coordinates="rtz",
-    data=["sqrt(g)_B", "|B|", "rho", "theta_B", "zeta_B"],
+    data=[
+        "sqrt(g)_Boozer_DESC",
+        "|B|",
+        "Boozer transform matrix",
+    ],
     M_booz="int: Maximum poloidal mode number for Boozer harmonics. Default 2*eq.M",
     N_booz="int: Maximum toroidal mode number for Boozer harmonics. Default 2*eq.N",
 )
 def _B_mn(params, transforms, profiles, data, **kwargs):
-    nodes = jnp.array([data["rho"], data["theta_B"], data["zeta_B"]]).T
-    norm = 2 ** (3 - jnp.sum((transforms["B"].basis.modes == 0), axis=1))
-    data["|B|_mn"] = (
-        norm  # 1 if m=n=0, 2 if m=0 or n=0, 4 if m!=0 and n!=0
-        * (transforms["B"].basis.evaluate(nodes).T @ (data["sqrt(g)_B"] * data["|B|"]))
-        / transforms["B"].grid.num_nodes
-    )
+    norm = data["Boozer transform matrix"]
+    data["|B|_mn_B"] = (
+        norm @ (data["sqrt(g)_Boozer_DESC"] * data["|B|"])
+    ) / transforms["B"].grid.num_nodes
+    return data
+
+
+@register_compute_fun(
+    name="R_mn_B",
+    label="R_{mn}^{\\mathrm{Boozer}}",
+    units="m",
+    units_long="meters",
+    description="Boozer harmonics of radial toroidal coordinate of a flux surface",
+    dim=1,
+    params=[],
+    transforms={"B": [[0, 0, 0]]},
```

Comment:
@rahulgaur104 if you want the zeta_B deriv of R you would do

transforms = {"B": [[0,0,1]]}

then have "R_mn" in your data

and in the compute, do `dR_dzeta_B = transforms["B"].transforms(data["R_mn"], dz=1)`